media/stylus/components/wiki/customcss.styl

```diff
@@ -791,144 +613,14 @@ ul.card-grid li {
 }
-
-/* ----------------------------
-        Styles for Indicator
-*/
-
-.headingWithIndicator {
-    clearfix();
-
-    h1, h2, h3, h4, h5, h6 {
-        float:left;
-    }
-}
-
-.inlineIndicator,
-.indicatorInHeadline,
-.blockIndicator,
-.overheadIndicator {
-/*box-shadow: 1px 1px 1px #CCC*/
-    margin: 0px 0px 1.5em;
-    border: 5px solid;
-    padding: 10px 10px 0px 10px;
-    background: none repeat scroll 0 0;
-    font-size: 12px;
-}
-.redesign .inlineIndicator,
-.redesign .indicatorInHeadline,
-.redesign .blockIndicator,
-.redesign .overheadIndicator {
-/* background-color:rgba-fallback(rgba(255,203,0,0.75); *)/
-    border-color: rgba-fallback(rgba(0,0,0,0.05));
-    border-style: solid !important;
-/* font-size: 11px; */
-    padding: 10px;
-}
-
-.blockIndicator,
-.overheadIndicator {
-    prevent-last-child-bottom-spacing();
-
-    pre {
-        color: $text-color;
-    }
-}
-
-.redesign .inlineIndicator,
-.redesign .indicatorInHeadline {
-    border-width: 0;
-    border-left-width: 4px;
-    font-size: 10px;
-    font-family: Helvetica, Arial, sans-serif;
-    min-width: 20px;
-    border-radius: 2px;
-    letter-spacing: 1px;
-    padding: 6px 6px 4px 6px;
-    display: inline-block;
-    vertical-align: top;
-    margin-left: 11px;
-    border-top-width: 0;
-    border-right-width: 0;
-    border-bottom-width: 0;
-    font-weight: normal;
-    line-height: normal;
-}
-/* styles for inline indicator */
-.inlineIndicator /* This class is base of inlineIndicator */,
-.megaWarningInline {
-    border-width: 1px;
-    border-style: solid;
-    font-size: 12px;
-    white-space: nowrap;
-    padding: 0 2px;
-    margin: 0 2px;
-}
-.todoInline {
-    border-left-width: 8px;
-    white-space: normal;
-}
-.megaWarningInline {
-    background: #f00;
-    border-color: #818151;
-    color: #fff;
-    font-size: small;
-    font-weight: 700;
-    float: right;
-    vertical-align: middle;
-/* Mega warnings: deep red backgrounds etc */
-}
-.notXPCOMInline,
-.noscriptInline {
-    font-weight: 700;
-}
-/* styles for overhead indicator */
-/*.overheadIndicator */
-/* This class is base of overheadIndicator */
-.standardNoteHeader,
-.standardNoteBlock {
-    border: 1px solid #000;
-    overflow: hidden;
-    margin: 10px 0;
-    padding: 0 10px;
-    text-align: center;
-}
-/*.overheadIndicator p,*/
-.blockIndicator p {
-    margin: 4px 0;
-}
-.draftHeader strong {
-    display: block;
-    padding-top: 10px;
-}
-.draftHeader div {
-    font-size: x-small;
-    padding-bottom: 10px;
-}
-html[dir="rtl"] .warningHeader {
-    padding: 1em 130px 1em 1em !important;
-    background-position: 99% 10px !important;
-}
-/* styles for Method indicator */
-.indicatorInHeadline,
-.standardNoteMethod {
-    border: 1px solid #000;
-    padding: 1px 2px;
-    white-space: nowrap;
-    vertical-align: middle;
-    font-size: 12px;
-    float: right;
-}
-[dir="rtl"] .indicatorInHeadline,
-[dir="rtl"] .standardNoteMethod {
-    float: left;
-}
+/* */
```

Comment:
There was a comment and I moved it into a different file but there still remained a logical separation between the code above what I removed and the code below it.

I would have used 
```/* -------------------------------------------------------------- */```
if I was in a file that had been re-orged but I wasn't so I chameleon coded instead.

Not sure there's a right answer here, happy to change it if you have a suggestion :)